src/pages/[platform]/tools/cli/index.mdx

```diff
@@ -0,0 +1,34 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'CLI',
+  description: 'CLI',
+  platforms: [
```

Comment:
eventually we'll probably want to move this to an import from `@/data`

Reply:
I think to get this to work we would need to relook at how generateDirectory.mjs works. At the moment it's just doing regex to get the meta object basically so it's not evaluating the javascript variable... Might have to do a follow up for this, what do you think?